tests/conftest.py

```diff
@@ -276,6 +276,25 @@ def remote_config_server_type(request):
     def server_type_legacy_grpc(request):
         return core.start_local_server(config=request.param, as_global=False)
 
+    @pytest.fixture(scope="session", params=[ServerConfig(protocol=CommunicationProtocols.gRPC,
+                                                          legacy=True),
+                                             ServerConfig(protocol=CommunicationProtocols.gRPC,
+                                                          legacy=False)],
+                    ids=[
+                        "ansys-grpc-dpf config",
+                        "gRPC CLayer config"
+                    ])
+    def remote_config_server_type(request):
```

Comment:
@cbellot000 You added it back on purpose? Also the source of the failing Codacy analysis.

Reply:
nope, by mistake